### DISCUSSION TITLE
How camera sensor works in mujoco?

### DISCUSSION THREAD

**Discussion by sanjay-swain** [google-deepmind/mujoco#1726] [General]
Hello, I am new to mujoco. I have previously worked on gazebo and there a separate sensor for camera was available. I have searched in the documentation and it seems like [camera](https://mujoco.readthedocs.io/en/latest/XMLreference.html#body-camera)  is under body but not under sensor? Is there any specific reason for this? Can you tell me if there are any differences on how they functions?

**Comment by yuvaltassa** [comment]
Sensors are floats computed by the engine on CPU at every timestep (default is 2ms), camera images are bytes computed by openGL on GPU, usually at a much lower rate (30 or 60 Hz). It doesn't really make sense to add them as sensors.

We've had discussions about this before, search the issues.
